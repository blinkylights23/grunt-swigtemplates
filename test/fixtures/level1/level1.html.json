{
	"local": "Level 1",
	"globalOverride": "Overridden"
}