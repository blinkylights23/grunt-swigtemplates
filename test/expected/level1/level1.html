Level 1
Inherited
Overridden
